### PR TITLE
Switch to multimodule maven, add web server

### PR DESCRIPTION
**Pull request by ronbun**

### CODE REVIEW

File: client/src/main/java/com/pk/lanserver/BasicUdpServer.java [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+@Slf4j
+public class BasicUdpServer implements UdpServer {
+  private String nick;
+  private String profileImg;
+  private DatagramSocket ds;
+  private Integer remotePort;
+  private CompletableFuture<List<Player>> futureActivePlayers = null;
+  private Instant start = null;
+  private Vector<Player> qActivePlayers = null;
```
Comment (Rogrir) [inline client/src/main/java/com/pk/lanserver/BasicUdpServer.java:35]: Why is this qActivePlayers and not just ActivePlayers?

File: client/src/main/java/com/pk/lanserver/LanTcpServer.java [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+      String localIp,
+      Integer localPort,
+      Integer remotePort,
+      String nick,
+      String profileImg,
+      Map<String, Connection> mapInvToConn)
+      throws IOException {
+    this.bQueueInvites = bQueueInvites;
+    this.bQueueMsgs = bQueueMsgs;
+    this.bQueueMoves = bQueueMoves;
```
Comment (Rogrir) [inline client/src/main/java/com/pk/lanserver/LanTcpServer.java:70]: b like binary?

Comment (ronbun) [reply client/src/main/java/com/pk/lanserver/LanTcpServer.java:70]: b like blocking

File: client/src/main/java/com/pk/lanserver/BasicUdpServer.java [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+import java.util.Vector;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Future;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class BasicUdpServer implements UdpServer {
+  private String nick;
+  private String profileImg;
+  private DatagramSocket ds;
```
Comment (Rogrir) [inline client/src/main/java/com/pk/lanserver/BasicUdpServer.java:31]: Don't like it as an 'ds' but let it be.

File: client/src/main/java/com/pk/lanserver/BasicUdpServer.java [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+    }
+    this.profileImg = profileImg;
+  }
+
+  /** */
+  @Override
+  public Integer call() throws Exception {
+    try {
+      ds.setSoTimeout(500);
+      for (; ; ) {
```
Comment (Rogrir) [inline client/src/main/java/com/pk/lanserver/BasicUdpServer.java:85]: what is this ; ; and why not ;;

File: client/src/test/java/com/pk/server/LanNetworkingTest.java [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class LanNetworkingTest {
+  private String test;
+  private String localAddr = "";
+  private Boolean flagFailed = false;
+  private BlockingQueue<Invite> bQIx = new LinkedBlockingQueue<>();
```
Comment (Rogrir) [inline client/src/test/java/com/pk/server/LanNetworkingTest.java:34]: I won't even comment those variables

File: client/src/main/java/com/pk/lanserver/BasicUdpServer.java (hunk 4 above)
Comment (ronbun) [reply client/src/main/java/com/pk/lanserver/BasicUdpServer.java:85]: Ask google java format, it's not my invention.

**Pull request merged**